### PR TITLE
OSSMDOC-424: Update supported CNI plugins service mesh

### PR DESCRIPTION
**Pull request by rh-tokeefe**
- Aligned team: Dev Tools
- For branches: 
- Jira: https://issues.redhat.com/browse/OSSMDOC-424
- Direct link to doc preview: https://deploy-preview-38223--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/preparing-ossm-installation.html#ossm-supported-configurations-networks_preparing-ossm-installation
- SME review: @
- QE review: @yxun 
- Peer review: @rolfedh 
- Cherry pick to 4.6, 4.7, 4.8, 4.9, 4.10

### CODE REVIEW

File: modules/ossm-supported-configurations.adoc [hunk 1]
```diff
@@ -33,6 +33,7 @@ For additional information about {ProductName} lifecycle and supported configura
 
 * OpenShift-SDN
 * OVN-Kubernetes is supported as a technology preview in {product-title} version 4.7.
+* Third-Party CNI Plug-ins that have been certified on {product-title} and passed {ProductShortName} conformance testing. See link:https://access.redhat.com/articles/5436171[Certified OpenShift CNI Plug-ins] for more information.
```
Comment (rolfedh) [inline modules/ossm-supported-configurations.adoc:36]:
```suggestion
* Third-party Container Network Interface (CNI) plug-ins that have been certified on {product-title} and passed {ProductShortName} conformance testing. See link:https://access.redhat.com/articles/5436171[Certified OpenShift CNI Plug-ins] for more information.
```

**Review by rolfedh — APPROVED** [review]

File: modules/ossm-supported-configurations.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-supported-configurations.adoc:36]: Done

**Comment by rh-tokeefe** [comment]
Dev and QE reviewed this content in the review meeting.

**Pull request merged**